caliban/docker.py

```diff
@@ -378,7 +378,7 @@ def _credentials_entries(user_id: int,
   return ret
 
 
-def _notebook_entries(lab: bool = False, version: Optional[str] = None) -> str:
+def _notebook_entries(lab: bool = False, version: Optional[str] = None, dlvm: bool = False) -> str:
```

Comment:
Can we make this flag called `scheduler` instead? I think it doesn't depend on `dlvm` and COULD be a separate flag.